docs/specs/moniker.yml

```diff
@@ -194,8 +206,14 @@ outputs:
         {"url":"/docs/a","moniker_group":"9d4e15fd"}
       ],
       "moniker_groups":{
-        "4667fedf":["netcore-2.0"],
-        "9d4e15fd":["netcore-1.0","netcore-1.1"]
+        "4667fedf":{
+          "monikers": ["netcore-2.0"],
+          "config_moniker_range": ">= netcore-2.0"
```

Comment:
`config` -> `configured`?

Reply:
the monikerRange from yaml header can also be treated as `configured`?